content/en/docs/tasks/configmap-secret/managing-secret-using-kubectl.md

```diff
@@ -199,4 +199,4 @@ kubectl delete secret db-user-pass
 
 - Read more about the [Secret concept](/docs/concepts/configuration/secret/)
 - Learn how to [manage Secrets using config files](/docs/tasks/configmap-secret/managing-secret-using-config-file/)
-- Learn how to [manage Secrets using kustomize](/docs/tasks/configmap-secret/managing-secret-using-kustomize/)
\ No newline at end of file
+- Learn how to [manage Secrets using kustomize](/docs/tasks/configmap-secret/managing-secret-using-kustomize/)
```

Comment:
Why is a change on this line needed?

Reply:
+1 to @Rishit-dagli point. Link seems to open fine for me as well , i think no changes needed in that and it seems like there is no difference in the changes?

Reply:
> Why is a change on this line needed?

This is an excellent question. The change is definitely not needed, and I am also wondering why it is there.

I edited the document using the online editor of GitHub and I definitely did not touch this line.

If you have a look at the history of this file, you can see that my commit is not the first one, where there is a whitespace change in the last line. Have a look at this earlier commit for example: https://github.com/kubernetes/website/commit/6f1eed2750a823b2abf0df383e92dab565df23d9

My gut feeling tells me that the new line at the end of the file (which is supposed to be there according to regular linux convenctions), is being removed and added again depending on what tool is being used for editing the file.

I can re-do the commit using a regular text editor so the last line stays untouched. I need to retrigger the jobs anyway, as @Kartin494 [asked me to do so](https://github.com/kubernetes/website/pull/37906#issuecomment-1318131845).